js/flightlog_fielddefs.js

```diff
@@ -177,7 +177,8 @@ var
             "DEBUG_ALTITUDE",
             "DEBUG_FFT",
             "DEBUG_FFT_TIME",
-            "DEBUG_FFT_FREQ"
+            "DEBUG_FFT_FREQ",
+            "GYRO_RAW"
```

Comment:
Maybe I'm wrong, but I think the order matters, you need to add the  DEBUG_FRSKY_D_RX element. This is how the firmware is defined:
```
    DEBUG_FFT_FREQ,
    DEBUG_FRSKY_D_RX,
    DEBUG_GYRO_RAW,
    DEBUG_COUNT
```
And maybe is better to maintain the name: DEBUG_GYRO_RAW in place of GYRO_RAW.